### PR TITLE
Update to latest versions of the test-retry and test-distribution plugin

### PR DESCRIPTION
**Pull request by jonnybbb**

### CODE REVIEW

File: gradle/verification-metadata.xml [hunk 1]
```diff
@@ -2322,6 +2322,11 @@
             <sha256 value="b35af3a5ffb1107e5e2d0b1a1c0ec8c5198c0691ff03600346efc439c632b63b" origin="Generated by Gradle because artifact wasn't signed"/>
          </artifact>
       </component>
+      <component group="org.gradle" name="test-retry-gradle-plugin" version="1.1.7">
```
Comment (wolfs) [inline gradle/verification-metadata.xml:2325]: Can you only keep the entry for the last version of the retry plugin?

Comment (jonnybbb) [reply gradle/verification-metadata.xml:2320]: removed - sorry I assumed wrongly old versions should stay in here.

**Comment by wolfs** [comment]
@jonnybbb Why is this PR against release?

**Comment by jonnybbb** [comment]
@wolfs there is no real need to go into release. The changes we did for the test-retry plugin (https://github.com/gradle/test-retry-gradle-plugin/pull/60/files#diff-4b43e8bcfe035c6cc7635d5645e82d2aR67) fixed a compatibility issue with gradle 6.6 (TestNG only) and a deprecation warning releated to the configuration cache feature (https://gradle.slack.com/archives/C012L0276KF/p1595342462336600). 
The test-distribution plugin does not have to go into release either - if you apply it you will benefit from a better performance (https://docs.gradle.com/enterprise/test-distribution-gradle-plugin/#history).

**Comment by wolfs** [comment]
We are not going to upgrade the versions on `release`. Replaced by https://github.com/gradle/gradle/pull/14025 for `master`.

**Pull request closed without merging**